src/sql-parser/tests/testdata/ddl

```diff
@@ -1508,14 +1508,14 @@ CreateCluster(CreateClusterStatement { name: Ident("cluster"), options: [Cluster
 parse-statement
```

Comment:
Can you add a test to this file with the new syntax to make sure everything roundtrips?

Reply:
Absolutely! Done.